contracts/wrapper/NameWrapper.sol

```diff
@@ -162,13 +173,13 @@ contract NameWrapper is
     ) public override {
         uint256 tokenId = uint256(keccak256(bytes(label)));
         address registrant = registrar.ownerOf(tokenId);
-
-        require(
-            registrant == msg.sender ||
+        if (
+            !(registrant == msg.sender ||
                 isApprovedForAll(registrant, msg.sender) ||
-                registrar.isApprovedForAll(registrant, msg.sender),
-            "NameWrapper: Sender is not owner or authorised by the owner or authorised on the .eth registrar"
-        );
+                registrar.isApprovedForAll(registrant, msg.sender))
+        ) {
+            revert UnauthorisedEthWrap(bytes32(tokenId), msg.sender);
```

Comment:
Can we just use `Unauthorised` here?

Reply:
I had it like that before, but I am logging the node. If I wanted to log it here, I would only have the labelhash. I could create the namehash here if that isn't too much overhead.

Reply:
It's just a keccak hash, so I'd say go for it.